recipes-core/systemd/systemd/229/0001-nspawn-don-t-check-binary-OS-tree-before-all-mounts-.patch

```diff
@@ -0,0 +1,75 @@
+From 0a95296b24d636377a58592f1a30eb4312e5d3d2 Mon Sep 17 00:00:00 2001
+From: Krisztian Litkey <krisztian.litkey@intel.com>
+Date: Tue, 8 Dec 2015 16:02:56 +0200
+Subject: [PATCH 1/2] nspawn: don't check binary/OS-tree before all mounts are
+ in place.
+
+Do not check if the binary to run is in place or if the given
+directory looks like an OS tree hierarchy *before* we have put
+all the requested bind- and overlay-mounts in place. It is
+possible that some of the mounts are responsible for pulling in
+the necessary bits and pieces for a successful check.
+
+Upstream-Status: Pending
```

Comment:
On Thu, 2016-03-10 at 07:11 -0800, Gustavo Sverzut Barbieri wrote:
> > +Upstream-Status: Pending
> 
> could you point to the URL? bug tracker, patchwork or mail list where
> this was proposed?

"Pending" means "pending internal decision whether we submit this
upstream", i.e. this patch has not been submitted upstream yet and
therefore there is no further information available that could be added
here.

I've said before that this is not an ideal situation and better should
be avoided and/or not be tolerated for long, but I agree with Mikko, we
have the patches in this state now and need to get this PR merged to
proceed, so +1 from me for merging.